src/Controls/src/Core/HandlerImpl/VisualElement.Impl.cs

```diff
@@ -116,7 +116,9 @@ Semantics IFrameworkElement.Semantics
 		internal Semantics SetupSemantics() =>
 			_semantics ??= new Semantics();
 
-		double IFrameworkElement.Width => WidthRequest;
-		double IFrameworkElement.Height => HeightRequest;
+		double IView.Width => WidthRequest;
+		double IView.Height => HeightRequest;
+
+		Thickness IView.Margin => Thickness.Zero;
```

Comment:
This is the only addition. We override it on `View` and we had this value on `Page`.